dev-docs/bidders/pubmatic.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: bidder
-title: Pubmatic
-description: Prebid Pubmatic Bidder Adaptor
+title: PubMatic
+description: Prebid PubMatic Bidder Adaptor
 
```

Comment:
Please add `prebid_1_0_supported : true` to the header so it appears in the 1.0 download option.

Reply:
I've added the flag